routes/books.js

```diff
@@ -0,0 +1,13 @@
+var express = require('express');
+var router = express.Router();
+var bookList = require('../controllers/books/index.js')
+var addBook = require('../controllers/books/new.js')
+
+/* GET home page. */
+router.get('/', bookList.books);
```

Comment:
Let's make this change to user too if we're surrounding by newlines.

Reply:
Made comments to add white lines to users on users pr